### ISSUE TITLE
error uploading templates to datastores with characters that need to be escaped

### ISSUE THREAD

**Issue by rubiojr** [rubiojr/esx#4]
For example, "datastore1 (3)"

**Comment by harryyeh** [comment]
Yes I noticed this issue - this seems to be more of Linux command line issue I was looking into this - there's a problem when you try to create directories as well
